.goreleaser.yaml

```
@@ -0,0 +1,42 @@
+before:
+  hooks:
+    - make dep
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    ldflags:
+      - -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.buildTime={{.Date}}
+    main: ./cmd/keepalived-exporter
+archives:
+  - name_template: "{{ .ProjectName }}_{{ .Version }}_{{- title .Os }}_{{ .Arch }}"
```

Based on the fact to follow other repos...

```suggestion
  - name_template: "{{ .ProjectName }}-{{ .Version }}.{{ .Os }}-{{ .Arch }}"
```